api/envoy/extensions/filters/http/grpc_json_transcoder/v3/transcoder.proto

```diff
@@ -225,10 +225,10 @@ message GrpcJsonTranscoder {
   // If true, try to match the custom verb even if it is unregistered. By
   // default, only match when it is registered.
   //
-  // According to the http template syntax<https://github.com/googleapis/googleapis/blob/master/google/api/http.proto#L226-L231>,
+  // According to the http template `syntax<https://github.com/googleapis/googleapis/blob/master/google/api/http.proto#L226-L231>`,
```

Comment:
Sorry,  the RST link syntax is [here](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages)